sim/firesim-lib/src/main/scala/passes/ILATopWiring.scala

```diff
@@ -63,18 +63,19 @@ class ILATopWiringTransform(dir: File = new File("/tmp/")) extends Transform {
     }
 
     //body
+    val probetriggers = 2
```

Comment:
Do you have a particular reason for changing the default to 2 (other than resource utilization?)

Reply:
Resource utilization, I've never needed to write a trigger of the form 'probe === a OR probe === b'. I think we should optimize for the common case.

Reply:
what about trying to trigger on combinations of ready && valid ? (without an explicit fire signal)

Reply:
Thats fine when `probetriggers = 2`.

What doesn't work is a trigger like
`count === 3'h1 || count === 3'h2`